src/sqs/backfill.test.ts

```diff
@@ -25,6 +24,41 @@ mocked(Sentry.Hub).mockImplementation(() => ({
 	captureException: sentryCaptureExceptionMock
 } as Sentry.Hub));
 
+const mockSentryGetClient = () => {
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
```

Comment:
Because any time a ts-ignore is there its going to draw eyes.... this is the existing mock and ts-ignore not newly added, i just moved it into a function since i need to re-use it. I did have a look into changing the way its mocked but was causing grief so decided to leave it for now